server/services/prisonerService.js

```diff
@@ -83,21 +94,19 @@ function createPrisonerService(nomisClientBuilder) {
     }
   }
 
-  async function getRoRelation(nomisClient, bookingId) {
+  async function getRoPersons(nomisClient, bookingId) {
     const relations = await getRoRelations(nomisClient, bookingId)
     if (!relations[0]) {
       throw new RoRelationshipError('No RO relationship')
     }
 
     const personIds = relations.filter(r => !isEmpty(r.personId))
 
-    if (personIds.length > 1) {
-      logger.warn(`Multiple RO relationships for bookingId: ${bookingId}`)
-    } else if (isEmpty(personIds)) {
+    if (isEmpty(personIds)) {
       throw new RoRelationshipError('No RO person identifier')
     }
 
-    return personIds.sort((a, b) => b.personId - a.personId)[0]
+    return personIds
   }
```

Comment:
we were picking the highest person id, then getting the delius id for this person, but there may not always be a delius ID so we now carry on with all the person ids and pick on e later